R/derive_param_clinbenefit.R

```diff
@@ -0,0 +1,256 @@
+#' Derive a Clinical Benefit Parameter
+#'
+#' Add a clinical benefit/disease control parameter to the input dataset.
+#'
+#' @details
```

Comment:
Please provide more details. E.g., which variables are populated and how, are observations created for subjects not in the input dataset, which date is chosen if there is more than one assessment with clinical benefit...

It should be possible to predict the output dataset from the documentation.